src/Converter/Feature2Mesh.js

```diff
@@ -257,14 +252,15 @@ function featureToPolygon(feature, options) {
             console.warn('Feature to Polygon: integer overflow, too many points in polygons');
             break;
         }
-        const color = getProperty('color', options, randomColor, geometry.properties);
+        const context = { globals, properties: () => geometry.properties };
+        const style = feature.style.drawingStylefromContext(context);
+
+        const color = toColor(style.fill.color);
```

Comment:
Maybe you don't need to create a new constant, since `color` is only used once at line 264 in `fillColorArray(colors, count, color, start);`.

Reply:
Also : a constant named `color` allready exists (at line 226).

Reply:
yes, I change it